.gitignore

```diff
@@ -0,0 +1,5 @@
+.vscode
+__pycache__
+McPy/
+
+McPyConfig.yaml
```

Comment:
you should add a newline at the end as github has some problems with it (probably can be ignored though)

Reply:
Done, thanks for the review